data.proto

```diff
@@ -147,3 +153,57 @@ message SimpleA2CTrainingRunConfig {
   MLPNetworkConfig actor_network = 4;
   MLPNetworkConfig critic_network = 5;
 }
+
+message MuZeroTrainingConfig {
+  uint32 model_publication_interval = 1;
```

Comment:
Maybe structure that a little. I'm sure we could extract and potentially share some that configuration.